frontend/src/components/tiles/InstanceCounter.vue

```diff
@@ -26,6 +27,11 @@ export default {
         counter: {
             required: true,
             type: Number
+        },
+        darkerGray: {
+            type: Boolean,
+            required: false,
+            default: false
```

Comment:
Not going to block on this, but driving the counter by a class with the `type` field in would have been cleaner here, rather than using inline styling on the `ff-counter`